browser/layout/partials/qualtrics-tag.html

```diff
@@ -0,0 +1,63 @@
+<!--BEGIN QUALTRICS SITE INTERCEPT-->
```

Comment:
Are there any docs/original hosting for this script that we can put in a comment?

Reply:
I'll add some later